docs/user/openstack/install_upi.md

```diff
@@ -145,14 +149,13 @@ sudo subscription-manager repos --disable=* # if not done already
 
 sudo subscription-manager repos \
   --enable=rhel-8-for-x86_64-baseos-rpms \
-  --enable=openstack-16-tools-for-rhel-8-x86_64-rpms \
-  --enable=ansible-2.9-for-rhel-8-x86_64-rpms \
+  --enable=openstack-16.2-deployment-tools-for-rhel-8-x86_64-rpms \
```

Comment:
do we need to enable ansible 2.10 or ansible-core is already available?

Reply:
ansible-core is available in appstreams, there is no repo yet for rhel 8 :(

Reply:
I updated the content, let me know what you think.